src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,77 @@
+import classNames from 'classnames';
+import React, { useContext, useEffect } from 'react';
+import { TodosContext } from '../TodosContext/TodosContext';
+
+type Props = {
+  completed: boolean,
+  id: number,
+  title: string,
+  userId: number,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  completed, id, title, userId,
+}) => {
+  const {
+    showModal,
+    currentTodoId,
+    handleShowModal,
+    handleChangeUser,
+    handleCurrentTodo,
+  } = useContext(TodosContext);
+
+  useEffect(() => {}, [currentTodoId]);
+
+  const handleClick = () => {
+    handleCurrentTodo(id);
+    handleChangeUser(userId);
+    handleShowModal();
+  };
+
+  return (
+    <tr
+      data-cy="todo"
+      className={classNames({
+        'has-background-info-light': showModal,
+      })}
+      key={id}
+    >
+      <td className="is-vcentered">{id}</td>
+      <td className="is-vcentered">
+        {completed ? (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        ) : ''}
```

Comment:
```suggestion
        {!!completed && (
          <span className="icon" data-cy="iconCompleted">
            <i className="fas fa-check" />
          </span>
        ) }
```
or you can write null, not empty string